src/resources/js/v2/rsvp-block.js

```diff
@@ -138,6 +135,35 @@ tribe.tickets.rsvp.block = {};
 		} );
 	};
 
+	/**
+	 * Handle the RSVP toggle for listing in public attendee list.
+	 *
+	 * @since TBD
+	 *
+	 * @param {event} e submission event
+	 */
+	obj.handleDisplayToggle = function( e ) {
+		e.preventDefault();
+
+		const $input = $( e.target );
+		const rsvpId = $input.data( 'rsvp-id' );
+		const checked = $input.prop( 'checked' );
+		const attendeeIds = $input.data( 'attendee-ids' );
+		const nonce = $input.data( 'opt-in-nonce' );
+		const $container = e.data.container;
```

Comment:
Should we save before using it or use it directly? (not sure)

Reply:
I don't understand what you mean. What are you suggesting to save?

Reply:
I'm good with leaving it set here